templates/base/head_navbar.tmpl

```diff
@@ -1,4 +1,4 @@
-<nav class="ui container" id="navbar" aria-label="{{.locale.Tr "aria.navbar"}}">
+<nav class="ui container secondary stackable main menu" id="navbar" aria-label="{{.locale.Tr "aria.navbar"}}">
```

Comment:
I think it shouldn't use `container` here (although it's from old code)

`ui container` means limited with (not full width).

The `main` also seems a no-op here.